addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties

```diff
@@ -44,11 +44,17 @@ exim.importLogFiles.choosefile.title = Select Input Type
 exim.importLogFiles.import.menu.label = Import URLs from Logs or raw Files...
 exim.importLogFiles.log.type.modsec2 = ModSecurity2 Logs
 exim.importLogFiles.log.type.zap = ZAP Messages
+exim.importPcap/Pcapng.choosefile.filter.pcap.description = Pcap file (*.pcap)
+exim.importPcap/Pcapng.choosefile.filter.pcapng.description = Pcap file (*.pcapng)
+
+exim.importPcap/Pcapng.topmenu.import = Import a PCAP or PCAPNG file
+exim.importPcap/Pcapng.topmenu.import.tooltip = Import a PCAP or PCAPNG file and add the HTTP(S) traffic inside to the sites tree and history panel.
```

Comment:
I'd suggest dropping the slash in the key name. Just let it be importpcap or import.pcap